station-dispenser-api-v0/src/main/java/net/modificationstation/stationapi/mixin/dispenser/block/DispenserBlockMixin.java

```diff
@@ -0,0 +1,62 @@
+package net.modificationstation.stationapi.mixin.dispenser.block;
+
+import net.minecraft.block.DispenserBlock;
+import net.minecraft.block.entity.BlockEntity;
+import net.minecraft.block.entity.DispenserBlockEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.world.World;
+import net.modificationstation.stationapi.api.StationAPI;
+import net.modificationstation.stationapi.api.item.CustomDispenseBehavior;
+import net.modificationstation.stationapi.api.item.DispenseEvent;
+import net.modificationstation.stationapi.api.item.DispenseUtil;
+import net.modificationstation.stationapi.impl.dispenser.DispenserInfoStorage;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Unique;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.Redirect;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.util.Random;
+
+@Mixin(DispenserBlock.class)
+public class DispenserBlockMixin {
```

Comment:
Mixin classes should be package-private (no modificator) so they're not visible.

Reply:
privated